cookbooks/platelike-boundary/doc/platelike-boundary.md

```diff
@@ -13,20 +13,20 @@ this. To make it slightly more interesting, we choose a $2\times 1$ domain in
 
 Like for many other things, has a set of plugins for prescribed velocity
 boundary values (see
-Sections&nbsp;[\[parameters:Boundary_20velocity_20model\]][1] and
-[\[sec:prescribed-velocity-boundary-conditions\]][2]). These plugins allow one
+Sections&nbsp; `sec:parameters:Boundary_20velocity_20model` and
```

Comment:
I am not super familiar with this myself, but I think we do need some kind of `{ref}` in front, right? Please check if you can remove the `Sections`  then.
```suggestion
Sections&nbsp; {ref}`parameters:Boundary_20velocity_20model` and
```